django/db/models/base.py

```diff
@@ -239,7 +239,8 @@ def __new__(cls, name, bases, attrs):
             else:
                 # .. and abstract ones.
                 for field in parent_fields:
-                    new_class.add_to_class(field.name, copy.deepcopy(field))
+                    new_field = copy.deepcopy(field)
```

Comment:
Is `field` a class or class instance here? 
Because if it is class, it will not create a copy without some tricks.